google-cloud-core/resources/META-INF/google-cloud-core.xml

```diff
@@ -62,12 +62,10 @@
   </extensions>
 
   <extensions defaultExtensionNs="com.google.gct.core">
-    <!-- For IJ Community Edition -->
-    <usageTracker implementationClass="com.google.cloud.tools.intellij.analytics.GoogleUsageTracker"
-                  key="Idea"/>
-    <!-- For IJ Ultimate Edition -->
-    <usageTracker implementationClass="com.google.cloud.tools.intellij.analytics.GoogleUsageTracker"
-                  key="idea"/>
+    <!-- Exclude Android Studio-->
```

Comment:
Looks good! How about adding a similar message from the javadoc comment here as well - i.e. 
> To exclude a platform, add a new usageTracker extension with a key equal to the platform key and the NoOpUsageTracker.

It might make it a bit faster to remember/make sure it works as blacklist?

Reply:
sounds good